### PR TITLE
Bug-1882726 update ssh access doc

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with:
https://bugzilla.redhat.com/show_bug.cgi?id=1882726

It focuses on adding some edits and minor changes to the existing procedure and output associated with accessing a VMI by using ssh. The defect has notes regarding the changes made to the doc.

### CODE REVIEW

**Comment by lmandavi** [comment]
@phoracek Requesting your review - for details please see the defect notes.

**Comment by lmandavi** [comment]
https://bug-1882726-ssh-ext-ip-addr--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-accessing-vm-consoles.html#virt-accessing-vm-consoles-cli

**Review by phoracek — APPROVED** [review]
/lgtm

**Comment by lmandavi** [comment]
These are the only changes that were made as part of this PR in the "updates to ssh access" commit:
https://github.com/openshift/openshift-docs/pull/26724/commits/46185e4f7a4a882e5530128ecf51e1259cfa1caf
(The changes were quite minor in scope).

The other files showing changes do not belong to this PR, they were addressed via a different PR:
https://github.com/openshift/openshift-docs/pull/26298 - and this PR is presently in QE review stage.

**Comment by lmandavi** [comment]
If it makes sense to wait for this PR #26298  to be merged first, then merge this PR, we can do that.

File: modules/virt-installing-virtio-drivers-installing-windows.adoc
Comment (adellape) [inline modules/virt-installing-virtio-drivers-installing-windows.adoc:26]: I think there's an extra space at the beginning of this line, though that wouldn't affect rendering.

**Review by adellape — APPROVED** [review]
LGTM after squashing to 1 commit. :+1:

**Comment by lmandavi** [comment]
@adellape -This commit in this PR:
https://github.com/openshift/openshift-docs/pull/26724/commits/3b44c9fd72328ea401d91aeb2fa8e756f148b911

was picked up from the PR that was just merged:
https://github.com/openshift/openshift-docs/pull/26298

I might want to decouple this commit because it was already included in the merged PR? Will connect with you to get your thought on this.

**Comment by lmandavi** [comment]
CP to 4.6, 4.7. Please see my note above.

File: _topic_map.yml
Comment (adellape) [inline _topic_map.yml:2553]: Conflict here.

File: _topic_map.yml
Comment (adellape) [inline _topic_map.yml:2562]: Another conflict.

File: virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
Comment (adellape) [inline virt/virtual_machines/virt-installing-qemu-guest-agent.adoc:9]: This heading should get an anchor/ID added.

File: virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
Comment (adellape) [inline virt/virtual_machines/virt-installing-qemu-guest-agent.adoc:16]: Extra `+` here can be removed.

**Review by adellape — CHANGES_REQUESTED** [review]

**Comment by vikram-redhat** [comment]
@lmandavi can this be closed?

**Comment by lmandavi** [comment]
@vikram-redhat - Yes, this GitHub issue can be closed. The content changes are being completed via a different PR. Thanks.

**Pull request closed without merging**
